src/terrain/clients/bags.clj

```diff
@@ -0,0 +1,47 @@
+(ns terrain.clients.bags
+  (:use [clojure-commons.error-codes]
+        [slingshot.slingshot :only [try+ throw+]])
+  (:require [clj-http.client :as http]
+            [cemerick.url :refer [url]]
+            [clojure.tools.logging :as log]
+            [terrain.util.config :as config]))
+
+(defn- bags-url
+  ([]
+   (bags-url []))
+  ([components]
+   (-> (apply url (config/bags-base-url) components)
+       (str))))
+
+(defn has-bags
+  [username]
+  (:body (http/head (bags-url [username]))))
```

Comment:
In cases where you're not expecting a response body, it's probably better to explicitly return `nil`. That allows readers to quickly determine that the function is being called for side-effects (in this case, either an exception is thrown or not).

Reply:
Should be fixed now, thanks.